src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -47,12 +47,21 @@ public function __construct(FileLinkFormatter $fileLinkFormatter = null)
     protected function describeRouteCollection(RouteCollection $routes, array $options = []): void
     {
         $showControllers = isset($options['show_controllers']) && $options['show_controllers'];
+        $showAliases = $options['show_aliases'] ?? false;
```

Comment:
Shouldn't we update all the other descriptors (JSON, Markdown, etc.) too?

Reply:
Good point! `show_controllers` is only leveraged by the text descriptor. `show_aliases` should behave the same IMO.

But it's a good idea to add aliases info to other descriptors. I'll do it in this PR since it's related.